### ISSUE TITLE
Ask Wii U for memory details

### ISSUE THREAD

**Issue by gudenau** [wiiudev/pyGecko#10]
Instead of using hard coded values for memory range checking, we should ask the Wii U about it's memory permissions.

**Comment by ryanrocks462** [comment]
discovered by cosmocortney : loading The Dev Application "ActMgr" (account manager) in loadiine, it calls a few iosu commands and is possible to poke at it afaik (both from cafe sdk 2.09 and 2.13 worked for me) as well as these others - https://mega.nz/#!jlkQSBCD!zHAdPbDUNw0cNRTl0grx9sGXc_eXeKGgRGMZdsVaSAc

**Comment by fakedcake** [comment]
So these are actually safe to use and won't mess with your NAND? I heard some pleople saying that DevKit rpx files should never be used on retail systems.

**Comment by ryanrocks462** [comment]
Yes they're safe. Just don't install the installable images (WUP files - 000.h3 etc) in the folders obviously :P but other than that, the rpx's etc are safe, in the link above I've only put the ones that work, Some Dev Applications such as "system_config_tool" don't work, I assume it's because of a permissions issue, or maybe even dependencies Retail units don't have, I could be wrong tho.
